src/components/inputTodo.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] you do not have to disable eslint rules instead you can define rules  in the eslintrc.json file